fastlane/spec/cli_tools_distributor_spec.rb

```diff
@@ -2,15 +2,16 @@
 
 describe Fastlane::CLIToolsDistributor do
   it "runs the lane instead of the tool when there is a conflict" do
-    require 'fastlane/commands_generator'
     ARGV = ["sigh"]
+    require 'fastlane/commands_generator'
```

Comment:
That was for: "Fix cli_tools_distributor_spec to be runnable on its own"

If you go to master and run `bundle exec rspec spec/cli_tools_distributor_spec.rb` you should find that it has a failure.